### ISSUE TITLE
Missing rexml/document in the new generated application

### ISSUE THREAD

**Issue by hurricup** [rails/rails#41891]
To reproduce:

1. Create an application with ruby-3.0.1 and rails 6.1.3.
2. run `bin/rails g scaffold Project name:string`
3. run `rake db:migrate`

Get error about missing `rexml/document`

This may be fixed with adding `rexml` to the `Gemfile` of application, because there is no longer default rexml gem. 
Shouldn't it be there by default?

**Comment by intrip** [comment]
I couldn't reproduce with `ruby-3.0.1` and Rails `6.1.3.1` on MacOSX

**Comment by hurricup** [comment]
Tested on Ubuntu 20.04.

I was wrong with command, seems RubyMine replaced it with: 
`/bin/bash -c "/usr/share/rvm/bin/rvm ruby-3.0.1@test-fix do /usr/share/rvm/rubies/ruby-3.0.1/bin/ruby /home/user/RubymineProjects/untitled132/bin/spring rails 'db:migrate'"`

Here is the full trace:
```
LoadError: cannot load such file -- rexml/document
/home/user/.rvm/gems/ruby-3.0.1@test-fix/gems/bootsnap-1.7.3/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:34:in `require'
/home/user/.rvm/gems/ruby-3.0.1@test-fix/gems/activesupport-6.1.3.1/lib/active_support/dependencies.rb:332:in `block in require'
/home/user/.rvm/gems/ruby-3.0.1@test-fix/gems/activesupport-6.1.3.1/lib/active_support/dependencies.rb:299:in `load_dependency'
/home/user/.rvm/gems/ruby-3.0.1@test-fix/gems/activesupport-6.1.3.1/lib/active_support/dependencies.rb:332:in `require'
/home/user/.rvm/gems/ruby-3.0.1@test-fix/gems/selenium-webdriver-3.142.7/lib/selenium/webdriver/firefox.rb:22:in `<main>'
/home/user/.rvm/gems/ruby-3.0.1@test-fix/gems/bootsnap-1.7.3/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:23:in `require'
/home/user/.rvm/gems/ruby-3.0.1@test-fix/gems/bootsnap-1.7.3/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:23:in `block in require_with_bootsnap_lfi'
/home/user/.rvm/gems/ruby-3.0.1@test-fix/gems/bootsnap-1.7.3/lib/bootsnap/load_path_cache/loaded_features_index.rb:92:in `register'
/home/user/.rvm/gems/ruby-3.0.1@test-fix/gems/bootsnap-1.7.3/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:22:in `require_with_bootsnap_lfi'
/home/user/.rvm/gems/ruby-3.0.1@test-fix/gems/bootsnap-1.7.3/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:31:in `require'
/home/user/.rvm/gems/ruby-3.0.1@test-fix/gems/activesupport-6.1.3.1/lib/active_support/dependencies.rb:332:in `block in require'
/home/user/.rvm/gems/ruby-3.0.1@test-fix/gems/activesupport-6.1.3.1/lib/active_support/dependencies.rb:299:in `load_dependency'
/home/user/.rvm/gems/ruby-3.0.1@test-fix/gems/activesupport-6.1.3.1/lib/active_support/dependencies.rb:332:in `require'
/home/user/.rvm/gems/ruby-3.0.1@test-fix/gems/webdrivers-4.6.0/lib/webdrivers/geckodriver.rb:72:in `<main>'
/home/user/.rvm/gems/ruby-3.0.1@test-fix/gems/bootsnap-1.7.3/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:23:in `require'
/home/user/.rvm/gems/ruby-3.0.1@test-fix/gems/bootsnap-1.7.3/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:23:in `block in require_with_bootsnap_lfi'
/home/user/.rvm/gems/ruby-3.0.1@test-fix/gems/bootsnap-1.7.3/lib/bootsnap/load_path_cache/loaded_features_index.rb:92:in `register'
/home/user/.rvm/gems/ruby-3.0.1@test-fix/gems/bootsnap-1.7.3/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:22:in `require_with_bootsnap_lfi'
/home/user/.rvm/gems/ruby-3.0.1@test-fix/gems/bootsnap-1.7.3/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:31:in `require'
/home/user/.rvm/gems/ruby-3.0.1@test-fix/gems/activesupport-6.1.3.1/lib/active_support/dependencies.rb:332:in `block in require'
/home/user/.rvm/gems/ruby-3.0.1@test-fix/gems/activesupport-6.1.3.1/lib/active_support/dependencies.rb:299:in `load_dependency'
/home/user/.rvm/gems/ruby-3.0.1@test-fix/gems/activesupport-6.1.3.1/lib/active_support/dependencies.rb:332:in `require'
/home/user/.rvm/gems/ruby-3.0.1@test-fix/gems/webdrivers-4.6.0/lib/webdrivers.rb:4:in `<main>'
/home/user/.rvm/gems/ruby-3.0.1@test-fix/gems/bootsnap-1.7.3/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:23:in `require'
/home/user/.rvm/gems/ruby-3.0.1@test-fix/gems/bootsnap-1.7.3/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:23:in `block in require_with_bootsnap_lfi'
/home/user/.rvm/gems/ruby-3.0.1@test-fix/gems/bootsnap-1.7.3/lib/bootsnap/load_path_cache/loaded_features_index.rb:92:in `register'
/home/user/.rvm/gems/ruby-3.0.1@test-fix/gems/bootsnap-1.7.3/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:22:in `require_with_bootsnap_lfi'
/home/user/.rvm/gems/ruby-3.0.1@test-fix/gems/bootsnap-1.7.3/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:31:in `require'
/home/user/RubymineProjects/untitled132/config/application.rb:7:in `<main>'
/home/user/.rvm/gems/ruby-3.0.1@test-fix/gems/bootsnap-1.7.3/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:23:in `require'
/home/user/.rvm/gems/ruby-3.0.1@test-fix/gems/bootsnap-1.7.3/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:23:in `block in require_with_bootsnap_lfi'
/home/user/.rvm/gems/ruby-3.0.1@test-fix/gems/bootsnap-1.7.3/lib/bootsnap/load_path_cache/loaded_features_index.rb:92:in `register'
/home/user/.rvm/gems/ruby-3.0.1@test-fix/gems/bootsnap-1.7.3/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:22:in `require_with_bootsnap_lfi'
/home/user/.rvm/gems/ruby-3.0.1@test-fix/gems/bootsnap-1.7.3/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:31:in `require'
/home/user/.rvm/gems/ruby-3.0.1@test-fix/gems/bootsnap-1.7.3/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:53:in `require_relative'
/home/user/RubymineProjects/untitled132/Rakefile:4:in `<main>'
/home/user/.rvm/gems/ruby-3.0.1@test-fix/gems/bootsnap-1.7.3/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:59:in `load'
/home/user/.rvm/gems/ruby-3.0.1@test-fix/gems/bootsnap-1.7.3/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:59:in `load'
/home/user/.rvm/gems/ruby-3.0.1@test-fix/gems/railties-6.1.3.1/lib/rails/commands/rake/rake_command.rb:20:in `block in perform'
/home/user/.rvm/gems/ruby-3.0.1@test-fix/gems/railties-6.1.3.1/lib/rails/commands/rake/rake_command.rb:18:in `perform'
/home/user/.rvm/gems/ruby-3.0.1@test-fix/gems/railties-6.1.3.1/lib/rails/command.rb:52:in `invoke'
/home/user/.rvm/gems/ruby-3.0.1@test-fix/gems/railties-6.1.3.1/lib/rails/commands.rb:18:in `<main>'
/home/user/.rvm/gems/ruby-3.0.1@test-fix/gems/bootsnap-1.7.3/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:23:in `require'
/home/user/.rvm/gems/ruby-3.0.1@test-fix/gems/bootsnap-1.7.3/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:23:in `block in require_with_bootsnap_lfi'
/home/user/.rvm/gems/ruby-3.0.1@test-fix/gems/bootsnap-1.7.3/lib/bootsnap/load_path_cache/loaded_features_index.rb:92:in `register'
/home/user/.rvm/gems/ruby-3.0.1@test-fix/gems/bootsnap-1.7.3/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:22:in `require_with_bootsnap_lfi'
/home/user/.rvm/gems/ruby-3.0.1@test-fix/gems/bootsnap-1.7.3/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:31:in `require'
/home/user/RubymineProjects/untitled132/bin/rails:5:in `<top (required)>'
/home/user/.rvm/gems/ruby-3.0.1@test-fix/gems/spring-2.1.1/lib/spring/client/rails.rb:28:in `load'
/home/user/.rvm/gems/ruby-3.0.1@test-fix/gems/spring-2.1.1/lib/spring/client/rails.rb:28:in `call'
/home/user/.rvm/gems/ruby-3.0.1@test-fix/gems/spring-2.1.1/lib/spring/client/command.rb:7:in `call'
/home/user/.rvm/gems/ruby-3.0.1@test-fix/gems/spring-2.1.1/lib/spring/client.rb:30:in `run'
/home/user/.rvm/gems/ruby-3.0.1@test-fix/gems/spring-2.1.1/bin/spring:49:in `<top (required)>'
/home/user/.rvm/gems/ruby-3.0.1@test-fix/gems/spring-2.1.1/lib/spring/binstub.rb:5:in `load'
/home/user/.rvm/gems/ruby-3.0.1@test-fix/gems/spring-2.1.1/lib/spring/binstub.rb:5:in `<top (required)>'
<internal:/usr/share/rvm/rubies/ruby-3.0.1/lib/ruby/3.0.0/rubygems/core_ext/kernel_require.rb>:85:in `require'
<internal:/usr/share/rvm/rubies/ruby-3.0.1/lib/ruby/3.0.0/rubygems/core_ext/kernel_require.rb>:85:in `require'
/home/user/RubymineProjects/untitled132/bin/spring:10:in `block in <main>'
<internal:kernel>:90:in `tap'
/home/user/RubymineProjects/untitled132/bin/spring:7:in `<main>'
(See full trace by running task with --trace)

Process finished with exit code 1
```

**Comment by hurricup** [comment]
Looks like this is selenium problem after all, and just a duplication of #41502
